python/src/_messages.py

```diff
@@ -0,0 +1,755 @@
+# generated by datamodel-codegen:
+#   filename:  Envelope.json
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Sequence
+
+from pydantic import BaseModel, ConfigDict, Field
+
+
+class ContentEncoding(Enum):
+    identity = "IDENTITY"
+    base64 = "BASE64"
+
+
+class MediaType(Enum):
+    text_x_cucumber_gherkin_plain = "text/x.cucumber.gherkin+plain"
+    text_x_cucumber_gherkin_markdown = "text/x.cucumber.gherkin+markdown"
+
+
+class Source(BaseModel):
+    model_config = ConfigDict(
+        extra="forbid",
```

Comment:
If a field gets added, will this prevent an application using an older version of messages from de-serializing a message created by an application using the newer version of messages?

Reply:
Reworked for forward compatibility